sdk/src/main/java/io/dapr/utils/Properties.java

```diff
@@ -35,6 +35,13 @@
    */
   private static final String DEFAULT_STRING_CHARSET = StandardCharsets.UTF_8.name();
 
+  /**
+   * API token for Dapr after checking system property and environment variable.
+   */
+  public static final Supplier<String> DAPR_API_TOKEN = () -> getStringOrDefault(
+      "dapr.api.token",
+      Constants.DAPR_API_TOKEN, null);

```

Comment:
This is the point I was trying to make. See how this constant is only used here and in the unit tests. It can be easily removed from the Constants class and be hardcoded here. The test class can have its own constant too.

If constants need to be referenced outside the class, it is better to have a contextualized class instead of a generic class called "Constants". This is why I am trying to stop adding to the Constants class.